### DISCUSSION TITLE
No way to view website homepage if I'm logged in

### DISCUSSION THREAD

**Discussion by ba32107** [Glench/ExtPay#231] [General]
Hi! This is a minor annoyance. If I'm logged in with my developer account to https://extensionpay.com/, there is no way to get to the home page. Sometimes I want to check the docs, or just read the FAQ. In this case, I either need to log out or open the site in incognito.

**Comment by Glench** [comment]
I hear you, but just so you know there's a link to the docs from the logged-in page.
